librosa/effects.py

```diff
@@ -568,3 +570,74 @@ def split(y, top_db=60, ref=np.max, frame_length=2048, hop_length=512):
 
     # Stack the results back as an ndarray
     return edges.reshape((-1, 2))
+
+
+def preemphasis(y, coef=0.97, zi=None, return_zf=False):
```

Comment:
how about using a verb (`preemphasize`) rather than a noun (`preemphasis`)?
granted, all the other effects (`remix`, `split`, `stretch`, `shift`, `trim`) are simultaneously verbs and nouns, but i think we should use verbs, in the spirit of sklearn's `fit`, `transform`, `predict`, etc.

Reply:
I don't have strong feelings about this -- most of our functions (but not all) are named for what they return (chroma, melspectrogram, etc).  I expect most people would search for this by the noun than by the verb, but that's just a guess.

Reply:
yeah, me neither. i hate to bikeshed names, so i'll let you choose. i just wanted to have this voiced out

Reply:
Ok.  I'll leave this as is then.